aldryn_events/tests/frontend/integration/pages/page.events.crud.js

```diff
@@ -0,0 +1,53 @@
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
+'use strict';
+/* global by, element, browser, expect */
+
+// #############################################################################
+// INTEGRATION TEST PAGE OBJECT
+
+var eventsPage = {
+    site: 'http://127.0.0.1:8000/en/',
+    mainElementsWaitTime: 12000,
+
+    // log in, log out
+    editModeLink: element(by.css('.inner a[href="/?edit"]')),
+    usernameInput: element(by.id('id_cms-username')),
+    passwordInput: element(by.id('id_cms-password')),
+    loginButton: element(by.css('.cms_form-login input[type="submit"]')),
+    userMenuOption: element.all(by.css(
+        '.cms_toolbar-item-navigation li')).first(),
+
+    cmsLogin: function (object) {
+        // object can contain username and password, if not set it will
+        // fallback to 'admin'
+        object = object || {};
```

Comment:
Ok, will do. Then it should be renamed from ``object`` to ``credentialsAndElements``